make/buf/all.mk

```diff
@@ -27,7 +27,9 @@ FILE_IGNORES := $(FILE_IGNORES) \
 	internal/buf/cmd/buf/cache/ \
 	internal/buf/cmd/buf/workspacetests/other/proto/workspacetest/cache/ \
 	internal/buf/internal/buftesting/cache/ \
-	internal/pkg/storage/storageos/tmp/
+	internal/pkg/storage/storageos/tmp/ \
+	**/crashers/*.output \
```

Comment:
I'd prefer no `**` in this expression, as I don't believe this works with `.dockerignore`. Even the `*.output` and `*.quoted` might not work, which is why we only use paths otherwise`.

Replace `**` with the actual paths to the directories.